### PR TITLE
📝 README - fixed broken link

### PR DESCRIPTION
**Pull request by jeoffreyfischer**
As per my conversation with @tiagov8, one of the link was broken because the brackets in the [name](url were wrongly typed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ This is a Gatsby generated site pulling data from:
 > Rules repository lives here: https://github.com/SSWConsulting/SSW.Rules.Content
 
 Merging changes to **main** on this repo will trigger:
-- a build/release of the **main** branch in Staging [(https://staging.ssw.com.au/rules/)](https://tfs365.com/rules or check the [latest staging workflow](https://github.com/SSWConsulting/SSW.Rules/actions/workflows/build-deploy-staging.yml) for the url).
+- a build/release of the **main** branch in Staging [https://staging.ssw.com.au/rules/](https://tfs365.com/rules) or check the [latest staging workflow](https://github.com/SSWConsulting/SSW.Rules/actions/workflows/build-deploy-staging.yml) for the url).
```
Comment (tiagov8) [inline README.md:74]: Hey @jeoffreyfischer 

Isn't it odd that the **text link** and the **actual link** are different? I think they should match

**Review by tiagov8 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jeoffreyfischer) [reply README.md:74]: I noticed it but thought it was intentional.
But I agree, the name and URL should match

**Comment by jeoffreyfischer** [comment]
As per my conversation with Matt W, referencing the Staging Environment URL is bad practice as it could change in the future.
Instead, we should reference a link to the Deployments which shows the latest Staging Environment URL.

**Task**
- [ ] replace the Staging Environment URL with a link to the [latest staging workflow](https://github.com/SSWConsulting/SSW.Rules/deployments/staging)

**Comment by wicksipedia** [comment]
its not that it is bad practice, but we have environments configured in the repo and it would be safer to reference that.... in case the staging url changes again

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**